.claude/scripts/create_tc_hybrid.py

```
@@ -0,0 +1,202 @@
+import subprocess
+
+
+def run_script(script_name: str, args: list) -> dict:
+    """Run another script and capture JSON output"""
+    script_dir = os.path.dirname(os.path.abspath(__file__))
+    script_path = os.path.join(script_dir, script_name)
+
+    cmd = ['python3', script_path] + args
+
+    try:
+        result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)